### PR TITLE
WP-488: Makeshift 500

### PR DESCRIPTION
**Pull request by eilinora**
makes a  pretty 500 page with links
screen shot 2017-12-19 at 1 12 54 pm

### CODE REVIEW

File: packages/mwp-app-route-plugin/src/route.js [hunk 1]
```diff
@@ -40,20 +271,20 @@ export const onPreResponse = {
...
+			}
 		},
 		state: {
-			failAction: 'ignore', // ignore cookie validation, just accept
-		},
+			failAction: "ignore" // ignore cookie validation, just accept
+		}
 	},
-	handler: getHandler(languageRenderers),
+	handler: getHandler(languageRenderers)
```
Comment (mmcgahan) [inline packages/mwp-app-route-plugin/src/route.js]: throw an error in the handler function in order to get your custom 500 response

```js
	handler: (request, reply) => {
		reply(new Error());
	}, //getHandler(languageRenderers)
```

screen shot 2017-12-19 at 1 12 54 pm

File: packages/mwp-app-route-plugin/src/route.js [hunk 2]
```diff
@@ -13,13 +13,241 @@ export const onPreResponse = {
...
-			return reply.continue();
+			return reply(`
+<!DOCTYPE html>
+<html data-reactroot="">
+
+<head>
+	<title data-react-helmet="true">Meetup</title>
+	<meta data-react-helmet="true" name="viewport" content="width=device-width, initial-scale=1" />
+	<meta data-react-helmet="true" http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<meta data-react-helmet="true" http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (mmcgahan) [inline packages/mwp-app-route-plugin/src/route.js]: might want to keep the markup as simple as possible - it doesn't need to be optimized for _anything_ other than fast render

File: packages/mwp-app-route-plugin/src/route.js [hunk 3]
```diff
@@ -13,13 +13,241 @@ export const onPreResponse = {
...
...,0,.05)}.flatpickr-current-month .numInputWrapper{width:6ch;width:7ch\0;display:inline-block}.flatpickr-current-month .numInputWrapper span.arrowUp:after{border-bottom-color:rgba(0,0,0,.9)}.flatpickr-current-month .numInputWrapper span.arrowDown:after{border-top-color:rgba(0,0,0,.9)}.flatpickr-current-month input.cur-year{background:transparent;-webkit-box-sizing:border-box;box-sizing:border-box;color:inherit;cursor:default;padding:0 0 0 .5ch;margin:0;display:inline-block;font-size:inherit;font-family:inherit;font-weight:300;line-height:inherit;height:auto;border:0;border-radius:0;vertical-align:initial}.flatpickr-current-month input.cur-year:focus{outline:0}.flatpickr-current-month input.cur-year[disabled],.flatpickr-current-month input.cur-year[disabled]:hover{font-size:100%;color:rgba(0,0,0,.5);background:transparent;pointer-events:none}.flatpickr-weekdays{background:transparent;text-align:center;overflow:hidden;width:100%;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-align:center;-webkit-align-items:center;-ms-flex-align:center;align-items:center;height:28px}span.flatpickr-weekday{cursor:default;font-size:90%;background:transparent;color:
```
Comment (mmcgahan) [inline packages/mwp-app-route-plugin/src/route.js]: eventually we'll want to use the real `<Footer>` from mup-web, which is more reason to move this markup to the consumer app

**Comment by eilinora** [comment]
Closing this PR for now then, as it requires working other areas of code

**Pull request closed without merging**